core/src/main/scala/com/intel/oap/execution/ColumnarShuffledHashJoinExec.scala

```diff
@@ -78,6 +78,11 @@ case class ColumnarShuffledHashJoinExec(
     "buildTime" -> SQLMetrics.createTimingMetric(sparkContext, "time to build hash map"),
     "joinTime" -> SQLMetrics.createTimingMetric(sparkContext, "join time"))
 
+  // build check
+  for (attr <- buildPlan.output) {
```

Comment:
maybe define as a new func buildCheck?

Reply:
ok, let me do some refining to these build check.